src/sentry/models/commitfilechange.py

```diff
@@ -1,10 +1,33 @@
+from typing import Any, Iterable, Optional
+
 from django.db import models
 
-from sentry.db.models import BoundedPositiveIntegerField, FlexibleForeignKey, Model, sane_repr
+from sentry.db.models import (
+    BaseManager,
+    BoundedPositiveIntegerField,
+    FlexibleForeignKey,
+    Model,
+    sane_repr,
+)
 
 COMMIT_FILE_CHANGE_TYPES = frozenset(("A", "D", "M"))
 
 
+class CommitFileChangeManager(BaseManager):
+    def get_count_for_commits(
+        self, commits: Iterable[Any], organization_id: Optional[int] = None
+    ) -> int:
+        """
+        Warning: Because `sentry_commitfilechange` has no `organization_id`
+        index, do not pass an `organization_id` if there are many commits.
+        """
```

Comment:
This isn't totally accurate. We have separate indexes on both `commit_id` and `organization_id`. The problem is that we then need to do a `BitmapAnd` across these indexes, and because some orgs have a lot of `commitfilechange` rows it ends up really expensive to do so. It ends up a lot cheaper to just skip this step and rely on just the `commit` index.